website/tutorial/basic/10-subscriptions.mdx

```diff
@@ -0,0 +1,617 @@
+---
```

Comment:
General structure comment:

I would put the Authentication and Subscriptions subjects in a separated "Bonus" section that would be optional to the project completion.

Besides that, I feel that Subscriptions should be after Pagination and filtering matters.

Reply:
as discussed the structure has been adjusted and subscriptions are now part of the advanced tutorial